frontend/libs/erxes-ui/src/modules/inputs/components/EmailField.tsx

```
@@ -80,26 +116,34 @@ export const EmailFieldsProvider = ({
+    emailValidationStatus,
+  );
   const setEmails = useSetAtom(emailsFamilyState(recordId));
   const setShowEmailInput = useSetAtom(showEmailInputFamilyState(recordId));
+
   useEffect(() => {
-    setEmails(emails);
+    setEmails(formattedEmails);
     return () => {
       setShowEmailInput(false);
     };
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)